src/algorithms/calorimetry/NeutronReconstructionConfig.h

```diff
@@ -0,0 +1,12 @@
+// Subject to the terms in the LICENSE file found in the top-level directory.
```

Comment:
I think this was in one of the other Config.h files, but I forget which one.  However, since I have removed this file (due to there being no parameters), it is no longer relevant.